src/EditorFeatures/CSharpTest2/Recommendations/StaticKeywordRecommenderTests.cs

```diff
@@ -278,7 +279,7 @@ public async Task TestNotAfterDelegate()
         [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
         public async Task TestNotBetweenUsings()
         {
-            await VerifyAbsenceAsync(AddInsideMethod(
+            await VerifyKeywordAsync(AddInsideMethod(
```

Comment:
❓ Why did this change?

Reply:
We should expect the `static` keyword to show here since it's inside a method. That said, test name needs to be changed too.

Reply:
I'm fairly certain the test was not originally intended to test behavior inside of methods. I would prefer to fix the test so it's testing the behavior of using directives in the correct location rather than what we have here.

Reply:
In this case, 👍